README.md

```diff
@@ -1 +1,23 @@
-# vairocana
\ No newline at end of file
+# Vairocana
+
+[![Python Version](https://img.shields.io/badge/python-3.12-blue)](https://www.python.org/)
+[![Django Version](https://img.shields.io/badge/django-5-green)](https://www.djangoproject.com/)
+[![PostgreSQL](https://img.shields.io/badge/postgres-16-blue)](https://www.postgresql.org/)
+
+## Description
+
+Provide a concise and informative overview of your project. Explain its purpose, key features, and what it aims to achieve.
+
+## Getting Started
+
+These instructions will help you get a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+List any software and dependencies required to run your project. For example:
+
+* Python 3.x
+* Django
+* PostgreSQL

```

Comment:
[change-request]
User won't need to have these dependencies. All they need will be `docker` and `docker-compose`.